bin/sandbox

```diff
@@ -93,16 +99,16 @@ unbundled bundle install --gemfile Gemfile
 
 unbundled bundle exec rake db:drop db:create
 
-unbundled bundle exec rails generate spree:install \
+unbundled bundle exec rails generate solidus:install \
   --auto-accept \
   --user_class=Spree::User \
   --enforce_available_locales=true \
   --with-authentication=false \
   --payment-method=none \
   $@
 
-unbundled bundle exec rails generate solidus:auth:install --auto-accept
-unbundled bundle exec rails generate ${extension_name}:install --auto-accept
+unbundled bundle exec rails generate solidus:auth:install --auto-run-migrations
+unbundled ../exe/${extension_name}
```

Comment:
I'm not sure if we still need to retain `unbundled` in this call. I tried running `../exe/solidus_starter_frontend`  directly from the sandbox directory and it seemed fine.

Reply:
Not sure either. I'd say to keep it until it gives us some problems, but up to you.